improver_tests/developer_tools/conftest.py

```diff
@@ -173,7 +173,7 @@ def probability_above_fixture():
 
 @pytest.fixture(name="probability_over_time_in_vicinity_above_cube")
 def probability_over_time_in_vicinity_above_fixture():
-    """Probability of max air temperature in 1H above threshold cube from UKV"""
+    """Probability of precipitation accumulation in 1H in vicinity above threshold cube from UKV"""
```

Comment:
Isn't this cube actually the precipitation accumulation in 15 minutes, rather than 1 hour?

Reply:
Well spotted! I've changed this now.